modules/programming/module_programming_winnowing/pyproject.toml

```
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "module_programming_winnowing"
+version = "0.1.0"
+description = "Programming assessment Winnowing module."
+authors = ["Marlon Bucciarelli <marlon.bucciarelli@tum.de>"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = "3.11.*"
+athena = {path = "../athena", develop = true}
```

This needs to be adapted to the new project structure:
```suggestion
athena = { path = "../../../athena", develop = true }
```